openapi/openapi/src/test/scala/endpoints/openapi/EndpointsTest.scala

```diff
@@ -63,26 +63,75 @@ class EndpointsTest extends AnyWordSpec with Matchers with OptionValues {
     Fixtures.toSchema(Fixtures.Enum.colorSchema.docs) shouldBe expectedSchema
   }
 
-  "Recursive types" in {
-    val recSchema =
-      Schema.Reference(
-        "Rec",
-        Some(Schema.Object(
-          Schema.Property("next", Schema.Reference("Rec", None, None), isRequired = false, description = None) :: Nil,
-          additionalProperties = None,
-          description = None,
-          example = None
-        )),
-        None
-      )
+  val recSchema =
```

Comment:
Ah sorry for that, I have a tendency to hit auto-format a lot. I'll revert it.

Reply:
Now that I look more into it, the indentation has changes because I moved it out of the test, because I want to reuse it for both the lazy tagged and lazy record version

Reply:
OK, I missed that :)